app/services/mixpanel_analytics.rb

```diff
@@ -0,0 +1,21 @@
+module MixpanelAnalytics
+  def requested_feed(owner:, repo:)
+    event = 'Fetched feed'
+    properties = { owner: owner, repo: repo }
+    track(event, properties)
+  end
+
+  private
+
+  def track(event, properties)
+    tracker.track(user_id, event, properties)
+  end
+
+  def tracker
+    @tracker ||= Mixpanel::Tracker.new(ENV['MIXPANEL_TOKEN'])
+  end
+
+  def user_id
+    request.remote_ip
```

Comment:
Mmm that's true. I really just kind of want to track how many users track a particular repo. I was thinking of ip because it sort of gave me a rough idea of user count, despite the inaccuracies. But I like your idea better. The track method does take an optional IP argument which I can pass in. I'll do that!